Gemfile

```diff
@@ -56,3 +56,9 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
+
+gem 'omniauth'
+gem 'omniauth_openid_connect'
+gem 'omniauth-cognito-idp'
```

Comment:
Do we need both dependencies?

Reply:
Yes... We tried removing the omniauth_openid_connect and it seemed to break our setup.